en/components/themes/themes-list/material.md

```diff
@@ -0,0 +1,40 @@
+### Material theme
+Material theme is the default theme for igniteUI for Angular
+
+in order to use it, include one of the following mixins:
+```scss
+@include igx-theme($default-palette);
+```
+
+For the `light` version of the `Material theme` you can use the following mixin: 
+```scss
+@include igx-light-theme($default-palette);
+```
+
+For the `Dark` version of the `Material theme` you can use the following mixin: 
+```scss
+@include igx-dark-theme($default-palette);
+```
+
+Both `igx-light-theme` and `igx-dark-theme` mixins extend the `igx-theme` mixin.
+We create those mixins just for your convenience, otherwise, if you want to use the `igx-theme` but with a dark palette and dark schema you have to manually do the following:
```

Comment:
```suggestion
We created those wrapper mixins just for your convenience, otherwise, if you want to use the `igx-theme` but with a dark palette and dark schema you have to do the following:
```